lib/hooks/post-build.js

```diff
@@ -0,0 +1,51 @@
+var existsSync  = require('fs').existsSync;
+var path        = require('path');
+
+var hooks = ['configure',
+  'setup',
+  'willDeploy',
+  'willBuild', 'build', 'didBuild',
+  'willPrepare', 'prepare', 'didPrepare',
+  'willUpload', 'upload', 'didUpload',
+  'willActivate', 'activate', 'didActivate',
+  'didDeploy',
+  'teardown'
+];
+
+var readConfig = function(options) {
+  var root = options.root;
+  var filePath = options.deployConfigFile;
+  var fullPath = path.join(root, filePath);
+  var deployTarget = options.deployTarget;
+
+  if (!existsSync(fullPath)) {
+    return {};
+  }
+
+  var fn = require(fullPath);
+
+  return fn(deployTarget).runOnPostBuild || null;
+};
+
+module.exports = function(ctx, result) {
```

Comment:
still need to pass ctx ( the `this` from the hook)

can't use something like `postBuild.call(this, result)` because I need access to the build env to load the proper config variable